state/protocol/badger/mutator_test.go

```diff
@@ -556,15 +564,20 @@ func TestExtendEpochTransitionValid(t *testing.T) {
 		require.Error(t, err)
 
 		// finalize block 3 so we can finalize subsequent blocks
+		// ensure an epoch phase transition when we finalize block 3
```

Comment:
```suggestion
		// finalize block 3 so we can finalize subsequent blocks
```

The phase transition happens when we finalize block 4 below